config/clients/js/patches/add-method-specific-attributes.patch

```diff
@@ -0,0 +1,118 @@
+diff --git a/api.ts b/api.ts
+index e45e6c2..260e0bc 100644
+--- a/api.ts
++++ b/api.ts
+@@ -759,6 +759,8 @@ export const OpenFgaApiFp = function(configuration: Configuration, credentials:
+       const localVarAxiosArgs = await localVarAxiosParamCreator.check(storeId, body, options);
+       return createRequestFunction(localVarAxiosArgs, globalAxios, configuration, credentials, {
+         [attributeNames.requestMethod]: "check",
```

Comment:
Not a big fan of this - but can't see what else we can do on a short notice - thanks for taking care of it!

Reply:
Yeah don't worry I hate it also 😆  

Given that mustache doesn't support comparison I'm not sure how we can improve. I didn't really spend any time digging into the openapi generator docs to see what is supported.

I did also think about potentially offload it into code rather than the patch (i.e. pass all args into a function that returns the right stuff to pass into the call) but I don't like the idea of writing code to work around the generator limitations